BasiliskII/src/Unix/video_x.cpp

```diff
@@ -44,6 +44,12 @@
 # include <pthread.h>
 #endif
 
+#define ENABLE_XRENDER
```

Comment:
This is a hack.

Reply:
I do agree, but I'd like a few people to try it -- before I have to add an auto*.* test which will take me more time than the patch itself :-) -- if some people report success, I'll add a commit to make it all lurvey and autobeeeep friendly.